gravitee-apim-console-webui/src/management/api/api-ng-navigation/api-ng-v1-v2-menu.service.ts

```diff
@@ -318,7 +318,7 @@ export class ApiNgV1V2MenuService implements ApiMenuService {
     if (this.constants.org.settings.alert?.enabled && this.permissionService.hasAnyMatching(['api-alert-r'])) {
       const alertEngineLicenseOptions = {
         feature: ApimFeature.ALERT_ENGINE,
-        context: UTMTags.CONTEXT_ENVIRONMENT,
+        context: UTMTags.CONTEXT_API_ANALYTICS,
```

Comment:
Is it expected to have a context related to API_ANALYTICS for an alert engine license option?

Reply:
> Is it expected to have a context related to API_ANALYTICS for an alert engine license option?

Yes it's related to the user position in the console when he request license upgrade